assets/launcher/lang/sklmessages_pl_PL.properties

```diff
@@ -2,7 +2,7 @@ lang.name=Polski
 lang.code=pl_PL
 
 general.issues=Zgłoś problem
-general.support=Wesprzyj nas na Patreonie
+general.support=Wesprzyj nas na Patronite
```

Comment:
Patronite to nie Patreon

Reply:
🤔 Zawsze myślałem, że Patreon to to samo co Patronite.
Poprawię na Patreon'ie

Reply:
Poprawna forma to "Patreonie", gdyż litera "n" nie jest niema.

Reply:
Poprawna forma to "Patreonie", gdyż litera "n" nie jest niema.

Reply:
Poprawna forma to "Patreonie", gdyż litera "n" nie jest niema :)

Reply:
Rozumiem. W takim razie zostawiam tak jak było.